Use the pre-installed Chrome driver when running on GitHub

darxriggs opened the pull request:
This makes sure that the Chrome driver version matches the Chrome version.

darxriggs commented:
@puneetbehl Have the checks been disabled temporarily (no GitHub actions have been run for this pull request)?

puneetbehl commented:
@darxriggs Yes, there were some changes in the Github Workflow. Could you please rebase your changes now?

darxriggs committed:
Use the pre-installed Chrome driver when running on GitHub

The GitHub virtual environment already provides a Chrome and a matching Chrome driver.

For installed software see:
https://github.com/actions/virtual-environments/blob/ubuntu20/20220207.1/images/linux/Ubuntu2004-Readme.md#environment-variables-1

For available environment variables see:
https://docs.github.com/en/actions/learn-github-actions/environment-variables

darxriggs commented:
@puneetbehl Now I managed it to properly integrate the tooling with the GitHub workflow.
